marklogic-data-hub-central/ui/e2e/cypress/support/pages/browse.tsx

```diff
@@ -461,6 +465,17 @@ class BrowsePage {
   }
 
 
+  //data export modal
+  getStructuredDataWarning() {
+    return cy.get('[data-testid="export-warning"]');
```

Comment:
can we try using cy.findByTestId.

Reply:
Not sure if the cypress has this function, the RTL does have.

Reply:
I am surprised you arent aware. We have imported RTL extension for cypress that lets us use data-testid and some other methods that we are familiar with in RTL, since couple of months.
Just type cy.find and you will see.

Reply:
ok. replaced to findByTestId.